### PR TITLE
improve(scripts): better logs and comments in collateral and identifier proposal scripts

### PR DESCRIPTION
**Pull request by pemulis**
Signed-off-by: John Shutt <john.d.shutt@gmail.com>

**Motivation**

Some small improvements to logs and comments in the proposal scripts.

**Summary**

Minor changes to logging, which required some more imported artifacts and methods in the identifier proposal script.

**Details**

n/a

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [x]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [x]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

**Issue(s)**

n/a

### CODE REVIEW

**Review by nicholaspai — APPROVED** [review]

**Review by daywiss — APPROVED** [review]

File: packages/core/scripts/identifier-umip/1_Propose.js [hunk 1]
```diff
@@ -2,13 +2,16 @@
...
 // ganache-cli --fork https://mainnet.infura.io/v3/d70106f59aef456c9e5bfbb0c2cc7164 --unlock 0x2bAaA41d155ad8a4126184950B31F50A1513cE25
-// Then execute the script as: truffle exec ./scripts/identifier-umip/1_Propose.js --network mainnet-fork --identifier USDETH --identifier ETHBTC from core
+// Then execute the script as: yarn truffle exec ./scripts/identifier-umip/1_Propose.js --network mainnet-fork --identifier USDETH --identifier ETHBTC from core
 
 // Use the same ABI's as deployed contracts:
 const { getTruffleContract } = require("../../index");
 const Governor = getTruffleContract("Governor", web3, "1.1.0");
 const IdentifierWhitelist = getTruffleContract("IdentifierWhitelist", web3, "1.1.0");
+const Finder = artifacts.require("Finder");
+const Voting = artifacts.require("Voting");
```
Comment (mrice32) [inline packages/core/scripts/identifier-umip/1_Propose.js:12]: nit: prefer the `getTruffleContract` syntax so we can explicitly specify the version and we don't depend on truffle's `artifacts.require`, which means we would need to run this inside of a truffle exec script.

It's also probably a good idea to bump the versions of all of these to "latest".

File: packages/core/scripts/identifier-umip/1_Propose.js [hunk 2]
```diff
@@ -70,6 +76,24 @@ async function runExport() {
...
   Identifiers Proposed`);
   console.table(identifierTable);
+
+  console.log("Transaction: ", txn?.tx);
+
+  const finder = await Finder.deployed();
+  const oracleAddress = await finder.getImplementationAddress(web3.utils.utf8ToHex(interfaceName.Oracle));
+  console.log(`Governor submitting admin request to Voting @ ${oracleAddress}`);
+
+  const oracle = await Voting.at(oracleAddress);
```
Comment (mrice32) [inline packages/core/scripts/identifier-umip/1_Propose.js:86]: nit: why not `Voting.deployed()`?

**Review by mrice32 — COMMENTED** [review]
Looks awesome -- a few minor comments.

File: packages/core/scripts/identifier-umip/1_Propose.js (hunk 2 above)
Comment (pemulis) [reply packages/core/scripts/identifier-umip/1_Propose.js:86]: No particular reason, just what was copied and pasted from elsewhere.

**Pull request merged**